demo/src/features/debug3d/mod.rs

```diff
@@ -1,53 +1,71 @@
-use crate::features::debug3d::extract::Debug3dExtractJob;
-use rafx::framework::{VertexDataLayout, VertexDataSetLayout};
-use rafx::nodes::ExtractJob;
-use rafx::nodes::RenderFeature;
-use rafx::nodes::RenderFeatureIndex;
-use std::convert::TryInto;
-
-mod debug3d_resource;
+rafx::declare_render_feature_mod!();
+rafx::declare_render_feature_renderer_plugin!();
+
+rafx::declare_render_feature!(Debug3DRenderFeature, DEBUG_3D_FEATURE_INDEX);
+
 mod extract;
-mod plugin;
 mod prepare;
 mod write;
-pub use plugin::Debug3DRendererPlugin;
 
-pub use debug3d_resource::*;
-use rafx::api::RafxPrimitiveTopology;
+mod public;
 
-pub fn create_debug3d_extract_job() -> Box<dyn ExtractJob> {
-    Box::new(Debug3dExtractJob::new())
-}
+use distill::loader::handle::Handle;
+use rafx::assets::MaterialAsset;
 
-pub type Debug3dUniformBufferObject = shaders::debug_vert::PerFrameUboUniform;
+pub use public::debug3d_resource::DebugDraw3DResource;
 
-/// Vertex format for vertices sent to the GPU
-#[derive(Clone, Debug, Copy, Default)]
-#[repr(C)]
-pub struct Debug3dVertex {
-    pub pos: [f32; 3],
-    pub color: [f32; 4],
+struct StaticResources {
```

Comment:
Would call it Debug3dStaticResources so we don't have N StaticResources structs

Reply:
Resolving to consolidate discussion of prefixes into https://github.com/aclysma/rafx/pull/122#discussion_r610704654.